src/ica_api.c

```diff
@@ -1237,6 +1239,7 @@ unsigned int ica_rsa_crt_key_check(ica_rsa_key_crt_t *rsa_key)
 
```

Comment:
OPENSSL_cleanse for tmp_buf before it is freed in line 1231? It dies contain private key parts.

Reply:
Done.